desktop/core/src/desktop/lib/view_util.py

```diff
@@ -31,7 +31,7 @@
 
 
 def big_filesizeformat(bytes):
-  if bytes is None or bytes is "":
+  if bytes == None or bytes == "":
```

Comment:
`is` None is correct, please read about `is` in Python

https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

Reply:
Got good knowledge. Thanks